cgsmiles/drawing_utils.py

```diff
@@ -135,20 +135,25 @@ def make_mapped_edges(graph, plain_edges):
             mapped_edges[fragid].extend(plain_edges[frozenset([idx, jdx])])
     return mapped_edges
 
-def make_node_pies(graph, pos, cgmapping, colors, use_weights=False):
+def make_node_pies(graph, pos, cgmapping, colors, outline=False, radius=0.2, linewidth=0.2, use_weights=False):
     """
     Generate the slices for the matplotlip pies used to draw nodes.
     """
     for idx, node in enumerate(graph.nodes):
         position = pos[node]
         fragids = graph.nodes[node].get('fragid', None)
-        if fragids and len(fragids) > 1:
-
+        wedgeprops=None
+        if use_weights and fragids and len(fragids) > 1 and graph.nodes[node].get('weight', 1) == 0:
```

Comment:
This if/else tree got *huge*. Either simplify it, or abstract branches to helper functions.
For example, set sane defaults (`{'edgecolor': 'black', 'linewidth': linewidth}`, `pie_colors = ['white', 'white']`), and overwrite/change those as needed

Reply:
MAINT Issue